rpcs3/Emu/Cell/Modules/cellPad.h

```diff
@@ -192,14 +192,25 @@ struct pad_info
 {
 	atomic_t<u32> max_connect = 0;
 	std::array<u32, CELL_PAD_MAX_PORT_NUM> port_setting{ 0 };
+	std::array<u32, CELL_PAD_MAX_PORT_NUM> reported_statuses{};
 
 	SAVESTATE_INIT_POS(11);
 
 	pad_info() = default;
 	pad_info(utils::serial& ar);
 	void save(utils::serial& ar);
+
+	u32 get_max_connect() const
+	{
+		return std::min<u32>(max_connect, CELL_PAD_MAX_PORT_NUM);
+	}
+
+	bool may_be_connected(u32 port_no) const
```

Comment:
I don't think the may_be is necessary.
From a cell perspective it's either true or not.
It doesn't matter what the game thinks here

Reply:
But this is an old result from thread, it does not reflect the state of cellPadGetData.

Reply:
Well then it's probably more fitting to say confirmed_connected or something like that

Reply:
No, because it is not a confirmation, it is a merely a cached result that may be inaccuarte if the game's thread prority is lower than 512. I renamed to "is_reportedly_connected".